### PR TITLE
Update Codeception #5827

### PR DESCRIPTION
**Pull request by weisswurstkanone**

### CODE REVIEW

File: tests/_support/Helper/DataType/TestDataHelper.php [hunk 1]
```diff
@@ -1187,8 +1187,9 @@ public function assertObjectsWithMetadata(Concrete $object, $field, Concrete $co
      */
     protected function assertObjectMetadataEqual($expected, $value)
     {
-        $this->assertInternalType('array', $expected);
-        $this->assertInternalType('array', $value);
+        // see https://github.com/sebastianbergmann/phpunit/commit/50ad7e1c4e74dce3beff17bf9c9f5a458cbe9958
+        $this->assertTrue(is_array($expected), "expected an array");
+        $this->assertTrue(is_array($value), "expected an array");
```
Comment (jdreesen) [inline tests/_support/Helper/DataType/TestDataHelper.php:1192]: `assertInternalType` got removed in PHPUnit 9, while PHPUnit 8 added `assertIsArray` for this type of check:

```suggestion
        $this->assertIsArray($expected);
        $this->assertIsArray($value);
```

<br>

But as it seems that you're still using PHPUnit 7, I'm not sure why this change is necessary at the moment.

Comment (weisswurstkanone) [reply tests/_support/Helper/DataType/TestDataHelper.php:1192]: In this case I would say let's try to keep it under our own control instead of relying on specialized assertion methods which are now or then depecrated or need an replacement and basically do the same. It is the only occurence.

Comment (weisswurstkanone) [reply tests/_support/Helper/DataType/TestDataHelper.php:1192]: TestDataHelper basically extends from Module which used this Asset codeception trait. But it does not implement it anymore.

```
namespace Codeception;

use Codeception\Exception\ModuleException;
use Codeception\Lib\Interfaces\RequiresPackage;
use Codeception\Lib\ModuleContainer;
use Codeception\Util\Shared\Asserts;

/**
 * Basic class for Modules and Helpers.
 * You must extend from it while implementing own helpers.
 *
 * Public methods of this class start with `_` prefix in order to ignore them in actor classes.
 * Module contains **HOOKS** which allow to handle test execution routine.
 *
 */
abstract class Module
{
  => use Asserts;
```

It seems that it in the past this was just delegated to

```
    protected function assertInternalType($type, $actual, $description = '')
    {
        \PHPUnit\Framework\Assert::assertInternalType($type, $actual, $description);
    }
```

which is not the case now. As said, I would prefer to be on the safe and flexible side :-)
